src/test/resources/3_1/security.yaml

```diff
@@ -0,0 +1,182 @@
+openapi: 3.1.0
+info:
+  title: security-test
+  version: 0.1.0
+servers:
+- description: Localhost Server
+  url: http://127.0.0.1:8000
+paths:
+  /token:
```

Comment:
Please delete this route and all component schemas that use it. The auth client should take card of handling this. I understand that your local server is vending this. The component security scheme already defines what is needed here.